app/controllers/users_controller.rb

```diff
@@ -1,6 +1,10 @@
 class UsersController < ApplicationController
   def show
-    @user = current_user
+    if current_user || current_admin
```

Comment:
Again, I'm not sure we need the || clause here. I assume it's here for a reason though.